[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-2 (#114273)
openshift/openshift-docs: security/compliance_operator/co-scans/compliance-scans.adoc

toc::[]

[role="_abstract"]

> The `ScanSetting` and `ScanSettingBinding` APIs are recommended to run compliance scans with the Compliance Operator.

Reviewer: ```suggestion
You can use the `ScanSetting` and `ScanSettingBinding` APIs to run compliance scans with the Compliance Operator.
```

> For more information on these API objects, run:

Reviewer: ```suggestion
For more information on these API objects, run the following commands:
```
